plugin/coin_policy_manager.go

```diff
@@ -0,0 +1,52 @@
+package plugin
+
+import (
+	"github.com/loomnetwork/go-loom"
+	ctypes "github.com/loomnetwork/go-loom/builtin/types/coin"
+	contract "github.com/loomnetwork/go-loom/plugin/contractpb"
+	"github.com/loomnetwork/loomchain"
+	"github.com/loomnetwork/loomchain/builtin/plugins/coin"
+	regcommon "github.com/loomnetwork/loomchain/registry"
+	"github.com/pkg/errors"
+)
+
+var (
+	// ErrCoinContractNotFound indicates that the Coin contract hasn't been deployed yet.
+	ErrCoinContractNotFound = errors.New("[CoinDeflationManager] CoinContract contract not found")
+)
+
+// CoinPolicyManager implements loomchain.CoinPolicyManager interface
+type CoinPolicyManager struct {
+	ctx   contract.Context
+	state loomchain.State
+}
+
+type Policy = ctypes.Policy
+
+// NewCoinPolicyManager attempts to create an instance of CoinPolicyManager.
+func NewCoinPolicyManager(pvm *PluginVM, state loomchain.State) (*CoinPolicyManager,
+	error) {
```

Comment:
keep `error)` in same line as function signature

Reply:
Done.